client/client.go

```diff
@@ -46,6 +46,8 @@ type Config struct {
 type TLSConfig struct {
 	SkipVerify bool
 	CA         string
+	MtlsCliCrt string
+	MtlsCliKey string
```

Comment:
Since this struct is `client.TLSConfig`, and a client certificate implies MTLS, these can be renamed to `Cert` and `Key`

Reply:
I try to differentiate from server cert and key by name. I can rename it put comment.

Reply:
Like I mentioned, the struct `client.TLSConfig` differentiates it from `server.TLSConfig` already

Reply:
👌